sdk/storage/storage-blob/src/utils/PooledBuffer.ts

```diff
@@ -0,0 +1,77 @@
+import { BuffersStream } from "./BuffersStream";
+import { Readable } from "stream";
+import { BLOCK_BLOB_MAX_STAGE_BLOCK_BYTES } from "./constants";
+
+const maxBufferLength = Math.min(
+  require("buffer").constants.MAX_LENGTH,
```

Comment:
>require("buffer") [](start = 2, length = 17)

We don't use require as it not optimized for es6 standards and bundlers. Use import instead.

Reply:
Can't use import as Typescript doesn't recognize "buffer".